src/app/vm/shared/vm.model.ts

```diff
@@ -21,120 +19,53 @@ export enum VmState {
 
 export const VmResourceType = 'UserVm';
 
-@ZoneName()
-@FieldMapper({
-  displayname: 'displayName',
-  serviceofferingid: 'serviceOfferingId',
-  serviceofferingname: 'serviceOfferingName',
-  securitygroup: 'securityGroup',
-  affinitygroup: 'affinityGroup',
-  zoneid: 'zoneId',
-  zonename: 'zoneName',
-  templateid: 'templateId',
-  templatename: 'templateName',
-  guestosid: 'guestOsId',
-  cpunumber: 'cpuNumber',
-  cpuspeed: 'cpuSpeed',
-  jobid: 'jobId',
-  cpuused: 'cpuUsed',
-  networkkbsread: 'networkKbsRead',
-  networkkbswrite: 'networkKbsWrite',
-  diskkbsread: 'diskKbsRead',
-  diskkbswrite: 'diskKbsWrite',
-  diskioread: 'diskIoRead',
-  diskiowrite: 'diskIoWrite',
-  keypair: 'keyPair',
-  isoid: 'isoId',
-  passwordenabled: 'passwordEnabled'
-})
-export class VirtualMachine extends BaseModel implements Taggable {
-  public static ColorDelimiter = ';';
-
-  public id: string;
-  public displayName: string;
-  public name: string;
-  public account: string;
-  public domain: string;
-  public domainid: string;
+export interface VirtualMachine extends BaseModelInterface, Taggable {
+  id: string;
+  displayname: string;
+  name: string;
+  account: string;
+  domain: string;
+  domainid: string;
   // Status
-  public state: VmState;
+  state: VmState;
   // Service Offering
-  public serviceOffering: ServiceOffering;
-  public serviceOfferingId: string;
-  public serviceOfferingName: string;
-  public cpuNumber: number;
-  public cpuSpeed: number;
-  public memory: number;
-  public volumes: Array<Volume>;
+  serviceOffering: ServiceOffering;
```

Comment:
Why all properties in lowercase except this one? Try to find usages and remove if necessary. This is not an API property.